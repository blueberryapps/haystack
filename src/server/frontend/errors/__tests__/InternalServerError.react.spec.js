import { shallow } from 'enzyme';
import React from 'react';
import InternalServerError from '../InternalServerError.react';

test('shallowly renders InternalServerError', () => {
  expect(shallow(<InternalServerError />)).toMatchSnapshot();
});
