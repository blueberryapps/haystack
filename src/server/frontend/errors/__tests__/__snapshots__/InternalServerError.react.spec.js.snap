exports[`test shallowly renders InternalServerError 1`] = `
<div>
  <h1>
    500: Haystack in fire.
  </h1>
  Run to 
  <a
    href="/">
    home
  </a>
  .
</div>
`;
